Practise_ClassWork/Human.cs

```diff
@@ -0,0 +1,27 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Practise_ClassWork
+{
+    public class Human
+    {
+       private readonly Fridge fridge;
+
+        public Human(Fridge fridge)
+        {
+           this.fridge = new Fridge();
+
+           this.fridge.EventFridge += CurrentFridgeInvoced;
+
+           this.fridge.Main();
+        }
+
+        public void CurrentFridgeInvoced(FridgeEvent e)
+        {
+            fridge.CurrentState();
```

Comment:
You don't use events at all.
please get familiar with the lection about events.

Reply:
-3 to events